tests/apps/haar_tree/main.c

```diff
@@ -171,20 +171,9 @@ int main(int argc, char *argv[])
             break;
         }
     }
-    pargv = malloc( (pargc+1) * sizeof(char*));
```

Comment:
Wait, we still need to find "--" on the command line to pass it to parsec_init: arguments before "--" are not understood by parsec_init and will create an error message.

Reply:
yes look at line 168 above